go.mod

```diff
@@ -0,0 +1,11 @@
+module github.com/Zedjones/teleirc
```

Comment:
This should be changed :)

Reply:
Whoopsie, good catch. That shouldn't be in there at all haha